pkg/rules/rules.go

```diff
@@ -79,14 +83,13 @@ func InputFromPaths(paths []string) (Input, error) {
 		go func(path string) {
 			defer wg.Done()
 
-			result, err := loader.RegoWithOpts(path, parse.ParserOptions())
```

Comment:
What this helper did for us isn't much, so I've inlined it below, added the indirection to read from fs.FS.